### ISSUE TITLE
Error 'TextEncoder' on EDGE

### ISSUE THREAD

**Issue by phanhuudinh** [adamgibbons/ics#129]
Ics not working on EDGE
my code:
```
...
const ics = require('ics');
...
```
Errors: 
```
ReferenceError: 'TextEncoder' is not defined
./node_modules/@hapi/address/lib/email.js
C:/Projects/InClound/Kona/kona-client/node_modules/@hapi/address/lib/email.js:11
   8 | 
   9 | var internals = {
  10 |   nonAsciiRx: /[^\x00-\x7f]/,
> 11 |   encoder: new (Util.TextEncoder || TextEncoder)() // $lab:coverage:ignore$
  12 | 
  13 | };
  14 | 
View compiled
__webpack_require__
C:/Projects/InClound/Kona/kona-client/webpack/bootstrap:786
  783 | };
  784 | 
  785 | // Execute the module function
> 786 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
      | ^
  787 | 
  788 | // Flag the module as loaded
  789 | module.l = true;
View compiled
fn
C:/Projects/InClound/Kona/kona-client/webpack/bootstrap:149
  146 | 		);
  147 | 		hotCurrentParents = [];
  148 | 	}
> 149 | 	return __webpack_require__(request);
      | ^
  150 | };
  151 | var ObjectFactory = function ObjectFactory(name) {
  152 | 	return {
View compiled
./node_modules/@hapi/address/lib/index.js
C:/Projects/InClound/Kona/kona-client/node_modules/@hapi/address/lib/index.js:5
__webpack_require__
C:/Projects/InClound/Kona/kona-client/webpack/bootstrap:786
  783 | };
  784 | 
  785 | // Execute the module function
> 786 | modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
      | ^
  787 | 
  788 | // Flag the module as loaded
  789 | module.l = true;
View compiled
```
I try with babel polyfill but still not work

**Comment by mindhaq** [comment]
This completely crashes a react app using this to create ICS dynamically (blank screen).

Microsoft Edge 44.18362.449.0

**Comment by adamgibbons** [comment]
Please let me know if this is fixed by #139
